index.js

```diff
@@ -541,7 +551,7 @@ const _bindWindow = (window, newWindowCb) => {
       if (context.isDirty()) {
         const windowHandle = context.getWindowHandle();
         nativeWindow.setCurrentWindowContext(windowHandle);
-        context.flush();
+        context.flush(); // TODO: Get rid of this; it causes a pipeline stall.
```

Comment:
Flush stalls? I thought finish does, not flush. But it’s a hint to OS X to commit raleslith tab frame buffers in sync.

It also seems unrelated to this pr.

Reply:
It seemed like I was getting higher FPS without it. Will do more testing.